changelog/README.md

```diff
@@ -5,6 +5,7 @@
 ### 4.3.0 (in progress)
 
 - [improvement] JAVA-2402: Add setTracing(boolean) to StatementBuilder
+- [bug] JAVA-2466: BatchStatement.newInstance is setting idempotence to false explicitly but should set to null
```

Comment:
As per our contribution guidelines, we should come up with a commit message that uses imperative style. e.g. "Set idempotence to null in BatchStatement.newInstance"